src/Query/Builder.php

```diff
@@ -890,6 +890,45 @@ public function decrementEach(array $columns, array $extra = [], array $options
         return $this->incrementEach($decrement, $extra, $options);
     }
 
+    /**
+     * Multiply a column's value by a given amount.
+     *
+     * @param  string     $column
+     * @param  float|int  $amount
+     * @return int
+     */
+    public function multiply($column, $amount = 1, array $extra = [], array $options = [])
```

Comment:
`$amount` should not have a default value. Multiplying by 1 is no-op, unless you have extra values.